### ISSUE TITLE
About RandomPatch

### ISSUE THREAD

**Issue by MagicXiaoJing** [VIS-VAR/LGSC-for-FAS#23]
![image](https://user-images.githubusercontent.com/5450325/85355941-3cc06e80-b540-11ea-8399-4376c40da9b6.png)
In paper, I see the great difference between the patch input and resize input. but in coding, I don't see the place you using random patch. Could you tell me whether you use random patch or not in the final?
Very thanks !!

**Comment by ZGSLZL** [comment]
Hi, we do use 'RandomPatch' in the paper, but not in FaceForensics Benchmark.

**Comment by MagicXiaoJing** [comment]
> Benchmark

Thanks. When using random patch in paper, it will be many subpatch from the face as input? And when inference, the face resize to 224x224 or the big image (larger than 224x224) as input?

**Comment by silvercherry** [comment]
> > Benchmark
> 
> Thanks. When using random patch in paper, it will be many subpatch from the face as input? And when inference, the face resize to 224x224 or the big image (larger than 224x224) as input?

Yes, I also meet this problem, if you do not use resize, then some pictures will exceed the picture size

**Comment by MagicXiaoJing** [comment]
perhaps training using random patch, evaluating using the resize.

**Comment by silvercherry** [comment]
> perhaps training using random patch, evaluating using the resize.

Have you solved the problem in training?When using randompatch, some pictures may be subpatched

**Comment by MagicXiaoJing** [comment]
what is subpatched ??

**Comment by silvercherry** [comment]
> what is subpatched ??

If you are using randompatch, start from the center of the face and add or subtract half of size, which sometimes exceeds the size of the entire picture. How do you handle it here

**Comment by MagicXiaoJing** [comment]
in that condition, give up randompatch, using orig img

**Comment by gehaocool** [comment]
When processing a high-resolution picture, the size of the face could exceed 224×224, so how to do the patching in this case?
Just patching part of the face, or using an input image bigger than 224×224？
And how do you get the "face images" from which to crop 224 patches?
